model/v1/span_test.go

```diff
@@ -0,0 +1,333 @@
+// Copyright (c) 2019 The Jaeger Authors.
```

Comment:
this also changed from jaeger. If we need to make changes we need to make them in jaeger first, such that this is a straight copy.